gradle/java.gradle

```diff
@@ -240,7 +228,9 @@ tasks.withType(Test).configureEach {
   timeout = Duration.ofMinutes(15)
 
   // Disable all tests if skipTests property was specified
-  enabled = !project.rootProject.hasProperty("skipTests")
```

Comment:
I don't think we use this property anywhere. Can we delete it?

Reply:
Good point - unlike maven, we have all of assemble, check, build to replace the need for this sort of property.